db/re/item_db_equip.yml

```diff
@@ -901,7 +901,7 @@ Body:
     EquipLevelMin: 40
     Refineable: true
     Script: |
-      bonus2 bAddEff,Eff_Curse,30;
+      bonus2 bAddEff,Eff_Curse,300;
```

Comment:
30 according to aegis data, do you know if/when it has been changed in the description?

Reply:
![Capture](https://user-images.githubusercontent.com/60462533/107880705-363d9780-6f13-11eb-857d-7e76be448630.PNG)